README.md

```diff
@@ -32,7 +32,7 @@ You can also create interaction model and publishing information for your Alexa
 As simple as:
 
 ```
-$ npx voxa create
+$ npx voxa-cli create
```

Comment:
let's hold on this change on the documentation. there should be a way to run it!

Reply:
why change from `voxa` to `voxa-cli` ?

Reply:
We found a problem on Windows when running `npx voxa create`. I'll make an update on the docs about it.